### PR TITLE
When a foriegn key is NULL and it's value is mapped with belongs_to, serialization breaks

### PR DESCRIPTION
**Pull request by lfglopes**
Fixes issue #476

### CODE REVIEW

File: lib/Serialization.php [hunk 1]
```diff
@@ -173,7 +173,11 @@ private function check_include()
 				try {
 					$assoc = $this->model->$association;
 
-					if (!is_array($assoc))
+					if (is_null($assoc))
```
Comment (lfglopes) [inline lib/Serialization.php]: changed. happy new year :)

**Comment by jpfuentes2** [comment]
Looks okay to me, how about you, @koenpunt ?

**Comment by koenpunt** [comment]
Sure, if @lfglopes can squash his commits into one that would be great

**Comment by jpfuentes2** [comment]
Agreed. Thanks, @lfglopes and @koenpunt.

**Comment by koenpunt** [comment]
@cvanschalkwijk we were waiting for a squash of those commits, to not pollute the commit history..

**Pull request merged**